### ISSUE TITLE
CT Cannot read property removeChild in phet-io-tests

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/energy-skate-park#67]
In CT for ESPB as well
```
energy-skate-park : phet-io-tests : assert
7 out of 8 tests passed. 1 failed.
SimTests: energy-skate-park: event indexing failed:
Uncaught TypeError: Cannot read property 'removeChild' of null
https://bayes.colorado.edu/continuous-testing/snapshot-1556260081150/phet-io-wrappers/js/tests/SimTests.js?bust=1556264539783:27


id: Bayes Chrome
Approximately 4/26/2019, 12:28:01 AM
energy-skate-park : phet-io-tests : assert
7 out of 8 tests passed. 1 failed.
SimTests: energy-skate-park: event indexing failed:
Uncaught TypeError: Cannot read property 'removeChild' of null
https://bayes.colorado.edu/continuous-testing/snapshot-1556260081150/phet-io-wrappers/js/tests/SimTests.js?bust=1556282187995:27


id: Bayes Chrome
Approximately 4/26/2019, 12:28:01 AM
energy-skate-park : phet-io-tests : no-assert
7 out of 8 tests passed. 1 failed.
SimTests: energy-skate-park: event indexing failed:
Uncaught TypeError: Cannot read property 'removeChild' of null
https://bayes.colorado.edu/continuous-testing/snapshot-1556260081150/phet-io-wrappers/js/tests/SimTests.js?bust=1556268950056:27


id: Bayes Chrome
Approximately 4/26/2019, 12:28:01 AM
```

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, locally I am also seeing
> r: Assertion failed: createTandem cannot accept dots: energySkatePark.labScreen.labModel.controlPoint.element~30.sourcePositionProperty

**Comment by jessegreenberg** [comment]
Now locally I am seeing
> Uncaught Error: Uncaught Error: Assertion failed: api mismatches present:

**Comment by jessegreenberg** [comment]
Tandems added were required so I must need to regenerate the schema. Then I should be able to see the original issue in that ticket. When I tree to generate the schema I see 
> $ grunt generate-phet-io-elements-files
Running "generate-phet-io-elements-files" task
Fatal error: Perennial task failed:
TimeoutError: Navigation Timeout Exceeded: 30000ms exceeded
  at Promise.then (C:\Users\Jesse\Documents\Development\phetsims\chipper\node_modules\puppeteer\lib\FrameManager.js:1276:21)
  at <anonymous>:null:null

I also had to add an entry called localTestingURL to build-local.json, and I don't know if I did that correctly.

**Comment by jessegreenberg** [comment]
@chrisklus showed me how to generate the schema so those errors are gone.

**Comment by jessegreenberg** [comment]
I haven't been able to see the original error yet because #68 and #69 are masking it for me locally. I am not hitting this with ?fuzz in studio. On hold until those issues are resolved.

**Comment by jessegreenberg** [comment]
#68 and #69 are no longer blocking this, I should be able to return to this.

**Comment by jessegreenberg** [comment]
PhET-iO Wrapper unit tests are all passing, this must have been fixed as part of #68 and #69. Thanks @zepumph!
